telos-frontend/src/views/journal/Journal.jsx

```diff
@@ -1,15 +1,64 @@
+import { useState } from 'react';
 import './Journal.css';
 import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
 import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import DatePicker from 'react-datepicker';
 import Page from './Page';
+import 'react-datepicker/dist/react-datepicker.css';
 
-const Journal = () => (
-  <div className="Journal">
-    <ArrowBackIcon className="Arrow-left" />
-    <Page />
-    <Page />
-    <ArrowForwardIcon className="Arrow-right" />
-  </div>
-);
+const Journal = () => {
+  const MILLISECONDS_PER_DAY = 1000 * 60 * 60 * 24;
+  const DAYS_TO_CHANGE_BY = 1;
```

Comment:
for the purpose of turning a page in a bullet journal, wouldn't we iterate 2 days at a time? Or is this the intended date iteration.

Reply:
Yeah that's a good question, at the moment when you turn the page, the two pages of the journal go forward/backward one day, so if a day's page was showing on the right half of the journal and the user clicks the right arrow, that day's page would shift to the left half of the journal. This made sense to me because no matter what day you're on, you can easily view between the 'yesterday' and 'today' pages without flipping back and forth between sets of pages all the time. But you are right in that when you flip the page in a physical journal, it skips to the next set of two pages. Might be worth having a discussion with the team about it, what do you think?.

Reply:
I see your point about always being able to see the next/prev day at the same time. I think that is really useful. IMO tho we should replicate real-life behaviour as its a bit funky having widgets shift from right to left.

I do think if we have time, it should be discussed. If we are time constrained, we could accept these changes, and note it down to discuss. If we change to an increment of 2, we can open a new issue.

Reply:
I definitely understand your reasoning, perhaps we can keep it like this for now, and do what you suggested: do a quick discussion and open a new issue if people prefer an increment of 2

Reply:
ok sounds good